src/server.c

```diff
@@ -5678,6 +5682,7 @@ sds genValkeyInfoString(dict *section_dict, int all_sections, int everything) {
             "io_threaded_writes_processed:%lld\r\n", server.stat_io_writes_processed,
             "io_threaded_freed_objects:%lld\r\n", server.stat_io_freed_objects,
             "io_threaded_poll_processed:%lld\r\n", server.stat_poll_processed_by_io_threads,
```

Comment:
I think we forgot to add the documentation about this here: https://github.com/valkey-io/valkey-doc/blob/main/commands/info.md?

Reply:
Done: https://github.com/valkey-io/valkey-doc/pull/165